### PR TITLE
Added hyperlink to wiki page for creating registrations [OSF-6037]

### PR DESCRIPTION
**Pull request by ml3ha**
## Purpose

To give users instruction on how registrations work if they have not yet registered yet.

## Changes

Added a hyperlink to the My Projects page for empty registrations.

## Side effects

On a local machine, the hyperlink takes the user to http://localhost:5000/myprojects/help.osf.io/#registrations rather than just help.osf.io/#registrations. I was not sure how to test this on my local machine; however, the hyperlink in the pull request is correct. 
## Ticket

https://openscience.atlassian.net/browse/OSF-6037

### CODE REVIEW

File: website/static/js/myProjects.js [hunk 1]
```diff
@@ -634,7 +634,8 @@ var MyProjects = {
                             'You have not created any projects yet.');
                     } else if (lastcrumb.data.nodeType === 'registrations'){
                         template = m('.db-non-load-template.m-md.p-md.osf-box',
-                            'You have not made any registrations yet.');
+                            'You have not made any registrations yet. Go to ',
+                            m('a', {href: 'help.osf.io/#registrations'}, 'Getting Started'), ' to learn how registrations work.' );
```
Comment (brianjgeiger) [inline website/static/js/myProjects.js]: Missing `http://` on this link. This tries to take you to `http://localhost:5000/myprojects/help.osf.io/#registrations` which is not a thing.

**Comment by brianjgeiger** [comment]
Pass done. 🐧

**Comment by ml3ha** [comment]
Updated the link with a leading http://

The hyperlink is working fine now on a local server

**Comment by brianjgeiger** [comment]
Looks great. 🎉 🐧 🎉

**Comment by brianjgeiger** [comment]
I cancelled Travis since this won't hit any tests that I know of.

**Pull request merged**